packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -380,8 +380,6 @@ abstract class ScrollableState<T extends Scrollable> extends State<T> {
   }
 
   void didUpdateScrollBehavior(double newScrollOffset) {
-    if (newScrollOffset == _scrollOffset)
```

Comment:
woah. That's a big change. Do we have an idea of how often this previously triggered in a good way that we're not doing all this extra work for? I think we used to assume that didUpdateScrollBehavior was essentially free.